source_pkg_noetic_core20/snap/snapcraft.yaml

```diff
@@ -0,0 +1,38 @@
+name: snapped-ros1-pkg
+base: core20 # the base snap is the execution environment for this snap
+version: '0.1' # just for humans, typically '1.2+git' or '1.3.2'
+summary: Basic ROS1 package snap # 79 char long summary
+description: |
+  Basic ROS1 package snap with config and launchfile based on core20
+grade: stable # must be 'stable' to release into candidate/stable channels
+confinement: strict # use 'strict' once you have the right plugs and slots
+
+parts:
+  snapped-ros1-pkg:
+    # See 'snapcraft plugins'
+    plugin: catkin
+    source: .
+    # catkin-packages:  # white list package to integrate
+    #   - snapped-ros1-pkg
+    build-packages:
+      - libboost-filesystem1.71.0
+      - liblog4cxx-dev
+      - libboost-regex1.71.0
+    stage-packages:
+      - ros-noetic-rosbash # necessary if we need rosrun
+      - ros-noetic-roslaunch # necessary if we need roslaunch
+apps:
+#  roscore:   # necessary only if you want to start the roscore from this snap
+#    command: opt/ros/noetic/bin/roscore
+#    plugs: [network, network-bind]
+#    extensions: [ros1-noetic]
```

Comment:
I'd remove that and instead leave a comment for the app `snapped-ros1-pkg` about the fact that it does not start the roscore unlike the app using a launch file.